acmwebsite/controllers/schedule.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+
+from icalendar import Calendar, Event
+from tg import expose, response
+
+from acmwebsite.lib.base import BaseController
+from acmwebsite.lib.helpers import log
+from acmwebsite.model import DBSession, Meeting
+
+
+class ScheduleController(BaseController):
+    def __init__(self):
+        self.meetings = DBSession.query(Meeting).order_by(Meeting.date).all()
+
+    @expose('acmwebsite.templates.schedule')
+    @expose('json', exclude_names=['page'])
+    def _index(self):
+        """Handle the schedule page."""
+
+        # Allow CORS
+        response.headers.add("Access-Control-Allow-Origin", "*")
+        return dict(page='schedule', meetings=self.meetings.filter(
+            Meeting.date > datetime.datetime.now() - datetime.timedelta(hours=3)
+        ))
+
+    @expose(content_type='text/calendar')
+    def acm(self):
+        cal = Calendar()
+        cal.add('prodid', '-//Mines ACM//web//EN')
```

Comment:
What is this hard coded prodid for? Can this be configured through paste config instead?

Reply:
Paste config? You mean in the `setup.py` file?

Reply:
No... the `.ini` file that configures paste... probably `development.ini` on your own machine.